gflownet/envs/base.py

```diff
@@ -242,8 +282,27 @@ def sample_actions(
             logits = policy_outputs
             logits /= temperature_logits
         if mask_invalid_actions is not None:
+            assert not torch.all(
+                mask_invalid_actions
+            ), """
+            All actions in the mask are invalid.
```

Comment:
Maybe wrap this in `textwrap.dedent` ? otherwise this will print weird indents:

```python
In [1]: print("""
   ...:             All actions in the mask are invalid.
   ...:             """)

            All actions in the mask are invalid.
            

In [2]: from textwrap import dedent

In [3]: print(dedent("""
   ...:             All actions in the mask are invalid.
   ...:             """))

All actions in the mask are invalid.

In [4]: repr("""
   ...:             All actions in the mask are invalid.
   ...:             """)
Out[4]: "'\\n            All actions in the mask are invalid.\\n            '"

In [5]: repr(dedent("""
   ...:             All actions in the mask are invalid.
   ...:             """))
Out[5]: "'\\nAll actions in the mask are invalid.\\n'"
```

Importantly the `repr` shows that the current `print` will leave trailing whitespaces that will affect the next one

Reply:
Or just remove the `"""` and make it a single line str :)

Reply:
Good point, thanks! I have wrapped the string with `dedent` in [this commit](https://github.com/alexhernandezgarcia/gflownet/pull/138/commits/3b6ff1fbef5e168c8bcd6fc8c71dc11cd197b49a).